src/containers/footer-links/footer-links.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Typography from '@material-ui/core/Typography';
+
+import { faInstagram, faFacebook } from '@fortawesome/free-brands-svg-icons';
+import { FOOTER_SOCIAL_NETWORK_LINKS } from '../../configs';
+
+import { useStyles } from './footer-links.styles';
+
+const FooterLinks = ({ language = 0 }) => {
```

Comment:
Don't use plain number for default value of parameter, use constant with explicit name

Reply:
Resolved